### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by aleemosman**
Change to Active Voice.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
```
Comment (brandyscarney) [inline CODE_OF_CONDUCT.md:19]:
```suggestion
community include the ability to:
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -18,10 +18,10 @@ diverse, inclusive, and healthy community.
...
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
+* Demonstrate empathy and kindness towards people
+* Be respectful of differing opinions, viewpoints, and experiences
 * Giving and gracefully accepting constructive feedback
```
Comment (brandyscarney) [inline CODE_OF_CONDUCT.md:23]:
```suggestion
* Give and gracefully accept constructive feedback
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -18,10 +18,10 @@ diverse, inclusive, and healthy community.
...
-* Accepting responsibility and apologizing to those affected by our mistakes,
+* Accept responsibility and apologize to those affected by our mistakes,
   and learning from the experience
```
Comment (brandyscarney) [inline CODE_OF_CONDUCT.md:25]:
```suggestion
  and learn from the experience
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -18,10 +18,10 @@ diverse, inclusive, and healthy community.
...
 * Focusing on what is best not just for us as individuals, but for the
```
Comment (brandyscarney) [inline CODE_OF_CONDUCT.md:26]:
```suggestion
* Focus on what is best not just for us as individuals, but for the
```

**Review by brandyscarney — CHANGES_REQUESTED** [review]

**Comment by liamdebeasi** [comment]
Merged. Thank you!

**Pull request merged**